conanfile.py

```diff
@@ -0,0 +1,52 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class QtProtobufConan(ConanFile):
+    name = "qtprotobuf"
+    version = "0.4.0"
+    license = "MIT"
+    url = "https://github.com/semlanik/qtprotobuf"
+    description = ("gRPC and Protobuf generator and bindings for Qt framework")
+    topics = ("conan", "qt", "protobuf")
+    settings = "os", "compiler", "build_type", "arch"
+    homepage = "https://github.com/semlanik/qtprotobuf"
+    options = {"shared": [True, False]}
+    default_options = {
+        "shared": False,
+        "qt:qtdeclarative": True,
+    }
+    generators = "cmake"
+    requires = [
+        "protobuf/3.9.1",
+        "protoc_installer/3.9.1@bincrafters/stable",
+        "qt/5.14.2@bincrafters/stable",
+        "grpc/1.25.0@inexorgame/stable",
```

Comment:
Not necessary, grpc is only dependency for tests and examples. Since you switched off both, lets remove this dependency.